boards/nucleo-f103/Makefile.features

```diff
@@ -1,4 +1,5 @@
 # Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_flashpage
```

Comment:
Nitpick: should be between `periph_cpuid` and `periph_gpio` (alphabetical order)